cmd/geth/main.go

```diff
@@ -362,11 +365,16 @@ func geth(ctx *cli.Context) error {
 	}
 
 	if ctx.Bool(utils.RunHeimdallFlag.Name) {
-		shutdownCtx, stop := signal.NotifyContext(context.Background(), os.Interrupt, syscall.SIGINT, syscall.SIGTERM)
+		_, stop := signal.NotifyContext(context.Background(), os.Interrupt, syscall.SIGINT, syscall.SIGTERM)
 		defer stop()
 
+		// TODO HV2: needs to be tested!
```

Comment:
Is this for running heimdall-v2 as a child process? In that case, can we expand the comment a bit?
It's anyway not used atm - not even in v1

Reply:
Yes, it is not tested at all. Changed the comment.